packages/block-library/src/post-featured-image/editor.scss

```diff
@@ -96,6 +96,13 @@
 		height: 100%;
 		width: 100%;
 	}
+
+	// When the Post Featured Image block is linked,
+	// it's wrapped with a disabled <a /> tag.
+	// Restore cursor style so it doesn't appear 'clickable'.
+	> a {
+		cursor: default;
```

Comment:
Note to self: I should probably also add this style to the image editor css after https://github.com/WordPress/gutenberg/pull/55470 🤔

Reply:
Thank you

Reply:
- https://github.com/WordPress/gutenberg/pull/55882